api/ssi-bridge/src/services/authentication-service.ts

```diff
@@ -63,7 +63,16 @@ export class AuthenticationService {
 			throw new Error('no jwt secret set!');
 		}
 
-		const signedJwt = jwt.sign({ user }, this.config.jwtSecret, { expiresIn: this.config?.jwtExpiration });
+		const jwtPayload = {
+			user: {
```

Comment:
@mastrogiovanni  shall I also remove the nesting into the `user` field? other props as `exp` and `iat` are for instance at root lvl. This would make existing JWT no more verifiable but this is not a big deal imo since one could simply generate a new one...

Reply:
But wouldn’t this change the decoding of the data of the jwt everywhere where we use the payload? E.g. ui-components would need to be adjusted